ibis/expr/tests/test_format.py

```diff
@@ -246,3 +246,25 @@ def output_type(self):
   second_arg:
     2.0"""
     assert result == expected
+
+
+def test_same_column_multiple_aliases():
```

Comment:
is this *only* a repr issue?

Reply:
Yes, it's definitely only a repr issue, because if you grab the columns from the object, they have the correct aliases